var/www/so/capme/.inc/callback-elastic.php

```diff
@@ -224,15 +224,15 @@ function cliscript($cmd, $pwd) {
 		}
 		$timestamp = $elastic_response_object["hits"]["hits"][0]["_source"]["@timestamp"];
 		$timestamp_epoch = strtotime($timestamp);
+		$orig_ts = $timestamp_epoch;
 		$mintime=time() - 5 * 365 * 24 * 60 * 60;
 		$maxtime=time() + 5 * 365 * 24 * 60 * 60;
 		if (filter_var($timestamp_epoch, FILTER_VALIDATE_INT, array("options" => array("min_range"=>$mintime, "max_range"=>$maxtime))) === false) {
 		        $errMsgElastic = "Invalid start time.";
 		}
 		// Set a start time and end time for the search to allow for a little bit of clock drift amongst different log sources
```

Comment:
Meant to comment on these previously but missed it.  Let's increase these values to 30 minutes before and 30 minutes after.

Reply:
Ok, are we adjusting $st_es and $et_es as well?

Reply:
`$st_es` gets computed based on `$st` (and likewise for `$et_es`), so we should just need to modify `$st` and `$et`.

Reply:
Sorry, that's what I meant -- I was a little confused, because the clip above shows `$mintime` and `$maxtime`, so I was under the impression that you want to adjust both `$st\$es` and `$mintime\$maxtime`